python/graphstorm/run/gsgnn_ep/ep_infer_lm.py

```diff
@@ -46,26 +46,32 @@ def main(config_args):
 
     gs.initialize(ip_config=config.ip_config, backend=config.backend,
                   local_rank=config.local_rank)
-
-    infer_data = GSgnnEdgeInferData(config.graph_name,
-                                    config.part_config,
-                                    eval_etypes=config.target_etype,
-                                    node_feat_field=config.node_feat_name,
-                                    label_field=config.label_field,
-                                    decoder_edge_feat=config.decoder_edge_feat)
+    # language model only encoder does not allow node and edge features
```

Comment:
Not understand what this comment wants to explain.

Reply:
Updated